cg/cli/workflow/balsamic/base.py

```diff
@@ -5,23 +5,21 @@
 import click
 from pydantic import ValidationError
 
-from cg.apps.balsamic.api import BalsamicAPI
-from cg.apps.balsamic.fastq import FastqHandler
-from cg.apps.environ import environ_email
-from cg.apps.hermes.hermes_api import HermesApi
-from cg.apps.housekeeper.hk import HousekeeperAPI
-from cg.apps.lims import LimsAPI
-from cg.apps.tb import TrailblazerAPI
-from cg.constants import EXIT_FAIL, EXIT_SUCCESS, Pipeline
-from cg.exc import AnalysisUploadError, BalsamicStartError, BundleAlreadyAddedError, LimsDataError
+from cg.cli.workflow.commands import link, resolve_compression, store_available
+from cg.constants import EXIT_FAIL, EXIT_SUCCESS
+from cg.exc import (
+    BalsamicStartError,
+    CgError,
+    DecompressionNeededError,
+)
 from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
-from cg.store import Store
+
 
 LOG = logging.getLogger(__name__)
 
 ARGUMENT_CASE_ID = click.argument("case_id", required=True)
 OPTION_DRY = click.option(
-    "-d", "--dry-run", "dry", help="Print command to console without executing", is_flag=True
+    "-d", "--dry-run", "dry_run", help="Print command to console without executing", is_flag=True
```

Comment:
```suggestion
    "-d", "--dry-run", help="Print command to console without executing", is_flag=True
```
This is not needed since `--dry-run` will be mapped to `dry_run` by click